### PR TITLE
add region names & 🔻post-Pochven🔻 updates

### PR DESCRIPTION
**Pull request by cdanis**

### CODE REVIEW

File: distance.sql [hunk 1]
```diff
@@ -1,28 +1,74 @@
...
+       b.security,
+       rb.regionName as destRegion,
+       MIN((a.x - b.x)*(a.x - b.x) + (a.y - b.y)*(a.y - b.y) + (a.z - b.z)*(a.z - b.z)) AS distance
 FROM mapSolarSystems AS a
 JOIN mapSolarSystems AS b
-WHERE b.security >= 0.0 AND ROUND(b.security, 1) < 0.5 AND a.security >= 0.0 AND ROUND(a.security, 1) < 0.5
--- Make sure destination system has a station
-AND (SELECT 1 FROM staStations as stations WHERE b.solarSystemID == stations.solarSystemID)
+LEFT JOIN mapRegions AS ra ON ra.regionId = a.regionId
+LEFT JOIN mapRegions AS rb ON rb.regionId = b.regionId
```
Comment (Calavoow) [inline distance.sql:41]: Is there a reason to care about the regions?

**Comment by Calavoow** [comment]
Who knew, SQLite can output an html table, neat

File: distance.sql (hunk 1 above)
Comment (cdanis) [reply distance.sql:41]: I'm needlehacking in Catch right now, and want to asset safety stuff back to k-space at some point but don't care about exactly where in Catch I do it from -- so wanted to see what my destination choices were.

Comment (cdanis) [reply distance.sql:41]: I think there's other use cases too -- "as Fortress Delve falls, where will the stuff mostly wind up?"

**Pull request merged**